recipes/pheasant/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pheasant" %}
+{% set version = "2.5.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0466da0a94eb2324dab43e4583b1251891adb012a6d264aaed42c7813cbfd07e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
It looks like you need to add an entry for a python entry_point based on:

https://github.com/daizutabi/pheasant/blob/master/setup.py#L90

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

and then add a command test:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands